plugin/src/main/java/io/fabric8/maven/plugin/ResourceMojo.java

```diff
@@ -236,6 +239,18 @@ public void executeInternal() throws MojoExecutionException, MojoFailureExceptio
     private void lateInit() {
         platformMode = clusterAccess.resolvePlatformMode(mode, log);
 
+        if (isOpenShiftMode()) {
+            Properties properties = project.getProperties();
+            if (!properties.contains(DOCKER_IMAGE_USER)) {
+                String namespace = clusterAccess.getNamespace();
+                log.info("Using docker image name of namespace: " + namespace);
+                properties.setProperty(DOCKER_IMAGE_USER, namespace);
+            }
+            if (!properties.contains(PlatformMode.FABRIC8_EFFECTIVE_PLATFORM_MODE)) {
```

Comment:
If I understand right, `FABRIC8_EFFECTIVE_PLATFORM_MODE` is used only as an internal marker for storing the detected mode when the platform mode is `auto` ?